docs/docs/cli-commands.md

```diff
@@ -436,7 +436,8 @@ Some generators require that their argument be a model in your `schema.prisma`.
 | `dataMigration <name>` | Generate a data migration component                                                                   |
 | `dbAuth`               | Generate sign in, sign up and password reset pages for dbAuth                                         |
 | `deploy <provider>`    | Generate a deployment configuration                                                                   |
-| `function <name>`      | Generate a Function                                                                                   |
+| `function <name>`      | Generate a Job                                                                                        |
```

Comment:
```suggestion
| `function <name>`      | Generate a Function                                                                                        |
```

Reply:
Bah, text wrapping confused me, should be fixed